example_workspace/.hlasmplugin/proc_grps.json

```diff
@@ -1,10 +1,15 @@
 {
   "pgroups": [
     {
-      "name": "P1",
-      "libs": [
-        "libs","libs2"
-      ]
+        "name": "P1",
+        "libs": [
+            "libs",
+            "libs2"
+        ],
+        "asm_options": {
+            "SYSPARM": "STH1",
+            "PROFILE": "STH2"
```

Comment:
I'd prefer if the example would actually made sense in the context of the sample workspace - so ideally a real profile macro that would utilize the `SYSPARM` value?

Reply:
Yes, I agree. I copied it simply from the story created in rally.